src/app/clusters/service-area-server/service-area-delegate.h

```diff
@@ -98,7 +90,9 @@ class Delegate
      * @brief The server instance ensures that the SelectedAreas and CurrentArea attributes are not null before
      * calling this method.
      * @param[in] skippedArea the area ID to skip.
-     * @param[out] skipStatusText text describing why current location cannot be skipped.
+     * @param[out] skipStatusText text describing why the current location cannot be skipped. Max size kMaxSizeStatusText.
+     * Note: If the return is false and skipStatusText is not successfully edited, example due to a string that can be longer than
+     * kMaxSizeStatusText, the size of this value should be set to 0 with .reduce_size(0) to avoid memory leaks.
```

Comment:
```suggestion
     * Note: If the return is false and skipStatusText is not successfully set, for example due to a string that can be longer than
     * kMaxSizeStatusText, the size of this value should be set to 0 with .reduce_size(0) to avoid callers using un-initialized memory.
```